src/components/ScreenWrapper/index.native.js

```diff
@@ -0,0 +1,30 @@
+import {React, useEffect} from 'react';
+import DevMenu from 'react-native-dev-menu';
+import compose from '../../libs/compose';
+import toggleTestToolsModal from '../../libs/actions/TestTool';
+import {propTypes, defaultProps} from './propTypes';
+import BaseScreenWrapper from './BaseScreenWrapper';
+import withEnvironment from '../withEnvironment';
+import withWindowDimensions from '../withWindowDimensions';
+
+const ScreenWrapper = (props) => {
+    useEffect(() => {
+        DevMenu.addItem('Open DevMenu', toggleTestToolsModal);
```

Comment:
Pardon me for my ignorance, but we don't need any env check here? `DevMenu` takes care of showing up in Dev/Emulator env only?

Reply:
Actually it does. https://github.com/zoontek/react-native-dev-menu/blob/master/src/index.ts

Reply:
thanks

Reply:
Can we call this "Open Test Preferences"?

Reply:
Shure!